library/src/main/java/com/airbnb/android/airmapview/WebAirMapViewBuilder.java

```diff
@@ -13,13 +17,42 @@
   }
 
   /**
+   * Set Mapbox Access Token for Web map
+   * @link https://www.mapbox.com/help/define-access-token/
+   * @param mapboxAccessToken Mapbox Access Token
+   * @return WebAirMapViewBuilder
+   */
+  public WebAirMapViewBuilder setMapboxAccessToken(String mapboxAccessToken) {
+    this.mapboxAccessToken = mapboxAccessToken;
```

Comment:
remove these setters and extract the token and ID from the `AirMapType options` passed in to `withOptions`